components/compliance-service/ingest/ingestic/mappings/comp-rep-date.go

```diff
@@ -228,6 +228,37 @@ var ComplianceRepDate = Mapping{
                     }
                   }
                 },
+                "refs": {
+                  "type": "nested",
+                  "properties": {
+                    "key": {
+                      "type": "keyword",
+                      "fields": {
+                        "engram": {
+                          "type": "text",
```

Comment:
is this right??  this is a copy/pasta job. i have no idea what i'm doing when it comes to es mappings

Reply:
You would only need the `engram` and `lower` fields if this value was going to be in the search bar or be used to filter on.

Reply:
ah cool, good to know, thank you! ya, no plans to include this in searchbar atm